submissions/kriLEBED/html_css_popup/css/style.css

```diff
@@ -0,0 +1,146 @@
+body {
+    font-family: 'Ubuntu', sans-serif; 
+    color: #858282;
+}
+li {
+    list-style-type: none;
+}
+a {
+    text-decoration: none;
+    color: #858282;
+}
+.wrapper{
```

Comment:
between the class name and the bracket should be a space. Could you please add all necessary alignments and spaces by code formatting? (the option at your code editor)